content/sensu-go/5.0/api/events.md

```diff
@@ -461,6 +462,65 @@ output               | {{< highlight json >}}
 }
 {{< /highlight >}}
 
+### `/events/:entity/:check` (PUT) {#eventsentitycheck-put}
+
+#### EXAMPLE {#eventsentitycheck-put-example}
+
+In the following example, an HTTP PUT request is submitted to the `/events/:entity/:check` API to create an event for the `sensu-go-entity` entity and the `check-cpu` check, resulting in a 200 (OK) HTTP response code.
+
+{{< highlight shell >}}
+curl -X PUT \
+-H "Authorization: Bearer TOKEN" \
+-H 'Content-Type: application/json' \
+-d '{
+  "entity": {
+    "entity_class": "agent",
+    "metadata": {
+      "name": "sensu-go-sandbox",
+      "namespace": "default"
+    }
+  },
+  "check": {
+    "interval": 60,
+    "metadata": {
+      "name": "check-cpu",
+      "namespace": "default"
+    }
+  }
+}' \
+http://127.0.0.1:8080/api/core/v2/namespaces/default/events/sensu-go-sandbox/check-cpu
```

Comment:
The resulting event from this request has a timestamp of `Timestamp: 1970-01-01 00:00:00 +0000 UTC`. Is there a way to use this endpoint like the [Sensu Core /request POST endpoint](https://docs.sensu.io/sensu-core/1.6/api/checks/#request-post)? Should we require a timestamp in the request body?